app/assets/stylesheets/_steps.scss

```diff
@@ -192,3 +203,37 @@ $dark-grey: #999999;
 .add-secondary-link--example-text {
   margin: 10px 0;
 }
+
+.step-by-step-list__table {
+  font-size: 16px;
+}
+
+.step-by-step-actions__list {
+  text-align: right;
+  width: 250px;
```

Comment:
This feels rather arbitrary, how does it look on mobile?

Reply:
removed - nice spot